pkgs/by-name/pd/pds/package.nix

```diff
@@ -0,0 +1,97 @@
+{
+  stdenv,
+  makeBinaryWrapper,
+  removeReferencesTo,
+  srcOnly,
+  python3,
+  pnpm,
```

Comment:
Most likely you need to change `pnpm` to `pnpm_9`. Since #371832, it points to `pnpm_10` instead of `pnpm_9`.

Reply:
Thanks! Would be painful to diagnose